src/sdam/server_description.ts

```diff
@@ -58,6 +60,7 @@ export class ServerDescription {
   minWireVersion: number;
   maxWireVersion: number;
   roundTripTime: number;
+  minRoundTripTime: number;
```

Comment:
Since this is public can we add documentation. Ideally we should have some for every field... but we can at least clarify what this is.